DOCS-13178- Add note to IPSec Tunnel section (#585)
mulesoft/docs-runtime-manager: modules/ROOT/pages/vpc-connectivity-methods-concept.adoc

image::vpc-overview-ipsec.png[IPsec Tunnel Overview]

[NOTE]

> If you are using the prior VPN solution (Legacy VPN), you need to migrate to Anypoint VPN. No further changes will be permitted on Legacy VPNs.

Reviewer: ```suggestion
If you are using a legacy VPN, migrate to Anypoint VPN before making changes. 
```
OR
```suggestion
If you are using the prior VPN solution (legacy VPN), you cannot make any changes until you migrate to Anypoint VPN. 
```
And a couple of questions: What "changes" are we referring to? Does this NOTE apply directly to using an IPsec tunnel, or does it apply to the topic set as a whole? The note text might belong in the topic set intro. I'd be happy to  edit surrounding texts when you make an addition or other change, if you have time: some of the legacy text could use work!

Author: Yes, the PM specifically asked the note be added under the IPSec Tunnel section.
